frameworks/helloworld/tests/test_sanity.py

```diff
@@ -29,12 +29,26 @@ def configure_package(configure_universe):
     try:
         sdk_install.uninstall(FOLDERED_SERVICE_NAME, package_name=PACKAGE_NAME)
 
-        sdk_upgrade.test_upgrade(
-            PACKAGE_NAME,
-            PACKAGE_NAME,
-            DEFAULT_TASK_COUNT,
-            service_name=FOLDERED_SERVICE_NAME,
-            additional_options={"service": {"name": FOLDERED_SERVICE_NAME}})
+        if shakedown.dcos_version_less_than("1.9"):
+            # Note: The hello-world package in DC/OS 1.8 Universe is 1.1.3 and doesn't
+            # have a /suppressed endpoint. It also defaults to user=root, whereas the
+            # following release will default to user=nobody (and this value can't change in an upgrade)
+            sdk_upgrade.test_upgrade(
```

Comment:
In #1372 @gabrielhartmann used `install` in this part of the if statement. That seems like an inconsistency, so I wouldn't mind some clarification as to which is the desired implementation.